study.md

```diff
@@ -96,5 +100,5 @@ If, in the above code, `var`, `let`, or `const` were removed and the code were
 executed as Ruby code, what type of object will be created?
 
 ```md
-<!-- your answer here -->
+instance object
```

Comment:
A hash with the keys "distance" and "solarMass" as symbols.  Their values are
25.04 and 2.135, respectively.